packages/console/src/pages/Connectors/index.tsx

```diff
@@ -110,10 +113,10 @@ const Connectors = () => {
               {!isLoading && !isSocial && (
                 <ConnectorRow connector={smsConnector} type={ConnectorType.SMS} />
               )}
-              {socialConnectors?.map((connector) => (
+              {socialConnectorGroups?.map(({ connectors }) => (
                 <ConnectorRow
-                  key={connector.id}
-                  connector={connector}
+                  key={connectors[0]?.id}
```

Comment:
This could be `undefined` from typescript code stand point, right? So why don't eslint complain about it?

Reply:
maybe undefined / null can be a "key" in React, anyway, updated with a new key.